CONTRIBUTING.md

```diff
@@ -4,11 +4,11 @@ Thanks for your interest in Noda Time. We appreciate all kinds of contributions,
 
 ## Basic Requirements
 
-If you want to contribute to the codebase, you're going to need a text editor or IDE. We recommend [Visual Studio Community 2017](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx) (Windows) or [Visual Studio Code](https://code.visualstudio.com/) (Windows/Linux/macOS).
+If you want to contribute to the codebase, you're going to need a text editor or IDE. We recommend [Visual Studio Community](https://visualstudio.microsoft.com/downloads/) (Windows), [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/), or [Visual Studio Code](https://code.visualstudio.com/) (Windows/Linux/macOS).
```

Comment:
Updated Visual Studio link and added link for Visual Studio for Mac.

Reply:
I'm not comfortable saying I *recommend* VS Code or VS for Mac for developing Noda Time, never having tried to do so myself. While they may be fine, I don't want to recommend something without experience. Something like "Visual Studio Code or Visual Studio for Mac should also work fine" would probably be better.

Reply:
Hmm, I didn't get the point. Do you want the Visual Studio for Mac to be removed ?

Reply:
From the *recommendation* list, yes. You can mention it in a separate sentence along the lines of "Visual Studio for Mac and JetBrains Rider should be fine as well, but we haven't tried developing Noda Time using them" if you want.